src/pls.cpp

```diff
@@ -391,9 +416,9 @@ Model& Model::plsr(const Mat2D& X, const Mat2D& Y, const METHOD algorithm) {
         }
         
         if (algorithm == KERNEL_TYPE1) {
-            t = X*r;
+            t = Xmat*r;
```

Comment:
X was used by the original paper.  Pls be reserved about changing any variable names in the algorithm itself.

Reply:
sure, made this change to avoid same-naming the stored X, but changed that to _X anyway - can change this back.